tests/installation/post_zdup.pm

```diff
@@ -27,6 +27,7 @@ sub run {
     # whatever the default in the image is.
     systemctl('set-default --force graphical.target');
 
+    script_run("rpm -qa --qf '%{vendor} %{name}\n' | tee /dev/$serialdev");
```

Comment:
what is this for? Seems like temporary debugging code?

Reply:
This is follow ludwig's code, I think it is for debug of vendor info of the packages.